### PR TITLE
[hive]Scala code format alarm clear #2835

### PR DESCRIPTION
**Pull request by huangxiaopingRD**
### What is the purpose of the change

Scala code format alarm clear in linkis-engineconn-plugins/hive

### Related issues/PRs

Related issues: #2835

### Brief change log

- Fix some Scala code formats and eliminate warnings
- change JavaConversions to JavaConverters

### Checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests).
- [x] I have explained the need for this PR and the problem it solves
- [x] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [Linkis mailing list](https://linkis.apache.org/community/how-to-subscribe) first)
- [ ] **If this is a code change**: I have written unit tests to fully verify the new behavior.

### CODE REVIEW

**Comment by casionone** [comment]
Code format check failed 
Run 'mvn spotless:apply' to fix these violations.

**Comment by huangxiaopingRD** [comment]
> Code format check failed Run 'mvn spotless:apply' to fix these violations.

It has been fixed, please help to review it again, thank you very much! @casionone

**Comment by casionone** [comment]
> > Code format check failed Run 'mvn spotless:apply' to fix these violations.
> 
> It has been fixed, please help to review it again, thank you very much! @casionone

Can I add your wechat friends? My wechat account:casinoxia

**Comment by huangxiaopingRD** [comment]
> > > Code format check failed Run 'mvn spotless:apply' to fix these violations.
> > 
> > 
> > It has been fixed, please help to review it again, thank you very much! @casionone
> 
> Can I add your wechat friends? My wechat account:casinoxia

sure

File: linkis-engineconn-plugins/hive/src/main/scala/org/apache/linkis/engineplugin/hive/executor/HiveEngineConnExecutor.scala [hunk 1]
```diff
@@ -298,20 +299,25 @@ class HiveEngineConnExecutor(
...
-      val scalaResult: mutable.Buffer[String] = result
+      val scalaResult: mutable.Buffer[String] = result.asScala
       scalaResult foreach { s =>
         val arr: Array[String] = s.split("\t")
         val arrAny: ArrayBuffer[Any] = new ArrayBuffer[Any]()
         if (arr.length > colLength) {
-          logger.error(s"""hive code 查询的结果中有\t制表符，hive不能进行切割,请使用spark执行""")
-          throw new ErrorException(60078, """您查询的结果中有\t制表符，hive不能进行切割,请使用spark执行""")
+          logger.error(
+            s"""There is a \t tab in the result of hive code query, hive cannot cut it, please use spark to execute"""
```
Comment (casionone) [inline linkis-engineconn-plugins/hive/src/main/scala/org/apache/linkis/engineplugin/hive/executor/HiveEngineConnExecutor.scala:308]: s"""There is a \t tab in the result of hive code query, hive cannot cut it, please use spark to execute(查询的结果中有\t制表符，hive不能进行切割,请使用spark执行)"""

File: linkis-engineconn-plugins/hive/src/main/scala/org/apache/linkis/engineplugin/hive/executor/HiveEngineConnExecutor.scala [hunk 2]
```diff
@@ -298,20 +299,25 @@ class HiveEngineConnExecutor(
...
+          )
+          throw new ErrorException(
+            60078,
+            """There is a \t tab in the result of your query, hive cannot cut it, please use spark to execute"""
+          )
```
Comment (casionone) [inline linkis-engineconn-plugins/hive/src/main/scala/org/apache/linkis/engineplugin/hive/executor/HiveEngineConnExecutor.scala:313]: """There is a \t tab in the result of your query, hive cannot cut it, please use spark to execute(您查询的结果中有\t制表符，hive不能进行切割,请使用spark执行)"""

**Review by casionone — APPROVED** [review]
LGTM

**Pull request merged**